poi-5.2.2/pom.xml

```diff
@@ -92,6 +92,7 @@
             org.apache.commons.collections4.*,
             org.apache.commons.math3.*,
             org.apache.commons.compress.*,
+            org.apache.logging.*,
```

Comment:
This one should be optional IMHO

Reply:
I am not sure, POI uses org.apache.logging. If it's not resolved, it will not work ?

Moreover, if the jar is packaged in this bundle like it is now, then pax logging doesn't work (because it tries to resolve an implementation of a logger in the classpath, which fails)